spec/support/contexts/payout_context.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+
+shared_examples 'payout that can create object events with #publish_created' do
+  let(:instance) { subject }
+
+  it {
+    initial_count = ObjectEvent.all.count
+    instance.publish_created
+    expect(ObjectEvent.all.count).to eq(initial_count + 1)
```

Comment:
I think this could be rewritten as:

```suggestion
expect {  instance.publish_created }.to change { ObjectEvent.count }.by(1)
```

Or something similar to that.

Reply:
Thanks. I felt like something like this existed, but could not recall for the life of me 😅